apps/server/src/schema/feedback/feedback-types.ts

```diff
@@ -0,0 +1,16 @@
+import { FeedbackTypeEnum } from '@repo/database';
+import { builder } from '../builder';
+
+export const FeedbackType = builder.enumType(FeedbackTypeEnum, {
+  name: 'FeedbackTypeEnum',
+});
+
+export const FeedbackDto = builder.prismaObject('Feedback', {
+  fields: (t) => ({
+    type: t.expose('type', {
+      type: FeedbackType,
+      nullable: false,
+    }),
+    message: t.exposeString('message', { nullable: false }),
```

Comment:
```suggestion
    message: t.exposeString('message', { nullable: false }),
    createdAt: t.expose('createdAt', { type: 'Date', nullable: false }),
    userId: t.exposeID('userId', { nullable: false }),
    user: t.expose('user', { nullable: false, type: UserDto }),
```
In general we are exposing all the things from schema. Consumer is the one that chooses what are the things that will actuall fetch.

Reply:
In the `user` field, let's use `t.relation` instead. 
`t.expose` won't work as it does not directly exists in the prisma model